data/Laxmankohar.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "Laxman Kohar",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Open Source Enthusiast || Web Developer || Aspiring Software Engineer",
+  "avatar": "https://github.com/Laxmankohar.png",
+  "links": [
+    {
+      "name": "Github",
+      "url": "https://twitter.com/Laxmankohar",
+      "icon": "FaGithub"
+    },
+    {
+      "name": "Twitter",
+      "url": "https://twitter.com/LaxmanKohar2",
+      "icon": "FaTwitter"
+    },
+    {
+      "name": "LinkedIn",
+      "url": "https://www.linkedin.com/in/laxman-kohar-a85a9a195",
+      "icon": "FaLinkedin"
+    },
+    {
+      "name": "Instagram",
+      "url": "https://www.instagram.com/laxmankohar92/",
+      "icon": "FaInstagram"
+    }
+    
```

Comment:
**Can you delete line number 28?**

Reply:
Don't worry, the Prettier GitHub Action will format the file